### PR TITLE
Align with ansible 2.4.2 compatibility

### PR DESCRIPTION
**Pull request by lukas-bednar**
Fixes: #39

### CODE REVIEW

**Comment by lukas-bednar** [comment]
The `inventory_file` was removed in 2.4 https://github.com/ansible/ansible/issues/32914
https://github.com/kubevirt/kubevirt-ansible/blob/a4af125c6ed5cd41657a4f3b3b3226f3b39d9d39/deploy-with-lago.yml#L15

**Comment by lukas-bednar** [comment]
@gbenhaim we will have to refactor that part related to inventory file in `deploy-with-lago.yml` file ...
Because this variable was removed in ansible 2.4 ... 

What exactly is purpose of following snippet ?
https://github.com/kubevirt/kubevirt-ansible/blob/a4af125c6ed5cd41657a4f3b3b3226f3b39d9d39/deploy-with-lago.yml#L13-L25

Do you think that it is possible to replace it with [add_host](http://docs.ansible.com/ansible/latest/add_host_module.html) module?

**Comment by gbenhaim** [comment]
It dynamically updates the inventory file with VMs that lago created.

**Comment by fabiand** [comment]
Anything that is missing?

**Comment by lukas-bednar** [comment]
@fabiand we need to align lago part https://github.com/kubevirt/kubevirt-ansible/pull/53#issuecomment-362646128

File: deploy-with-lago.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - hosts: localhost
   connection: local
-  gather_facts: False
+#  gather_facts: False
```
Comment (lukas-bednar) [inline deploy-with-lago.yml]: I will put this back

**Comment by lukas-bednar** [comment]
Now `inventory_dir` is not defined because `localhost` is not in inventory, and when I add it, then plays which has `hosts: all` are being executed also on localhost :-(

**Comment by lukas-bednar** [comment]
I decided to move lago initialization out of playbook because of `inventory_file` was removed in 2.4 and `inventory_dir` is defined only for hosts which are in inventory file. and localhost is not there.

@gbenhaim please review

**Comment by lukas-bednar** [comment]
I rebased it on latest changes

File: deploy-with-lago.yml (hunk 1 above)
Comment (lukas-bednar) [reply deploy-with-lago.yml]: ops

**Comment by lukas-bednar** [comment]
@gbenhaim Hi, I addressed your comments https://github.com/kubevirt/kubevirt-ansible/pull/53#issuecomment-364761205
Could you please take a look ? Thank you.

**Pull request merged**